portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  height: 100%; 
 }
 
-#greeting-container {
-  margin-top: 20px;
+.wrapper {  
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 800px;
 }
+
+.welcome {
```

Comment:
I don't see this in the HTML, is it unused?

Reply:
I deleted the HTML and forgot about the CSS - removed it